src/openrct2/ride/coaster/meta/FlyingRollerCoaster.h

```diff
@@ -54,6 +56,8 @@ constexpr const RideTypeDescriptor FlyingRollerCoasterAltRTD =
     SET_FIELD(AvailableBreakdowns, (1 << BREAKDOWN_SAFETY_CUT_OUT) | (1 << BREAKDOWN_RESTRAINTS_STUCK_CLOSED) | (1 << BREAKDOWN_RESTRAINTS_STUCK_OPEN) | (1 << BREAKDOWN_VEHICLE_MALFUNCTION) | (1 << BREAKDOWN_BRAKES_FAILURE)),
     SET_FIELD(LiftData, { SoundId::LiftBM, 4, 6 }),
     SET_FIELD(UpkeepCosts, { 49, 20, 90, 11, 3, 15 }),
+    SET_FIELD(DefaultPrices, { 20, 20 }),
+    SET_FIELD(DefaultMusic, MUSIC_STYLE_ROCK),
```

Comment:
Used to be ```    {   20, 20, MUSIC_STYLE_GENTLE,            },  // RIDE_TYPE_FLYING_ROLLER_COASTER_ALT```

Reply:
> The "alt" types of the Lay-down, Flying and multi-dim coasters have their default music style set to that of the "real" counterpart instead of gentle.